modular_skyrat/modules/customization/game/objects/items/conversion_kits.dm

```diff
@@ -0,0 +1,21 @@
+/obj/item/device/custom_kit
+    name = "modification kit" 
+    desc = "A box of parts for modifying a certain object."
+    icon = 'modular_skyrat/master_files/icons/donator/obj/kits.dmi'
+    icon_state = "partskit"
+    var/from_obj //The base object to be converted
+    var/obj/item/to_obj //The object to turn it into
```

Comment:
Might as well make it autodoc-compatible. Also, let's make both as items for now, we can change that later if need be, but might as well make it coherent.
```suggestion
    /// The base object to be converted.
    var/obj/item/from_obj
    /// The object to turn it into.
    var/obj/item/to_obj
```